docs/production-install.rst

```diff
@@ -48,37 +45,32 @@ Then the repository definition:
   enabled=1
   EOF
 
+Install the Socorro repository.
+::
+  sudo rpm -ivh https://s3-us-west-2.amazonaws.com/org.mozilla.crash-stats.packages-public/el/7/noarch/socorro-public-repo-1-1.el7.centos.noarch.rpm
```

Comment:
Eventually we'll need to make proper CNAMES for these sorts of things (especially considering that we're likely to move to a different zone).